### PR TITLE
Improve firewall rules clean up and doc

### PR DESCRIPTION
**Pull request by myrevery**
Clean up before saving iptables rules restore file, zeroing counts and avoiding persistent fail2ban ban lists, it should be managed by fail2ban, hope clean freaks are happy (like me:zany_face:).

Updated documentation for firewall to add custom script usage.

Upgrade backup now also include iptables rules restore file.

### CODE REVIEW

File: bin/v-stop-firewall [hunk 1]
```diff
@@ -12,7 +12,6 @@
 
 # Defining absolute path for iptables and modprobe
```
Comment (jaapmarcus) [inline bin/v-stop-firewall]: Remove comment " and modprobe"

**Review by jaapmarcus — APPROVED** [review]

**Comment by jaapmarcus** [comment]
I had to merge separate commits to the 1.6.4 staging branch as the docs changes will *** every thing up

https://github.com/hestiacp/hestiacp/tree/staging/1.6.14

As PHP8.2-imagick issues should also be resolve know we will aim for a release for next Tuesday

**Comment by jaapmarcus** [comment]
I also updated the comments slightly btw as you probably noticed...

**Comment by myrevery** [comment]
Thanks, hard work!

**Pull request merged**